src/OutputFormat.ts

```diff
@@ -0,0 +1,33 @@
+import {Writer} from './IonWriter';
+import {
+    makePrettyWriter,
+    makeBinaryWriter,
+    makeTextWriter
+} from './Ion';
+
+export enum OutputFormat {
+    PRETTY = "pretty",
+    TEXT = "text",
+    BINARY = "binary",
+    EVENTS = "events",
+    NONE = "none"
+}
+
+export namespace OutputFormat {
+    export function createIonWriter(name: OutputFormat) : Writer {
+        switch (name) {
+            case OutputFormat.PRETTY:
+                return makePrettyWriter();
+            case OutputFormat.TEXT:
+                return makeTextWriter();
+            case OutputFormat.BINARY:
+                return makeBinaryWriter();
+            case OutputFormat.EVENTS:
+                return makePrettyWriter();
+            case OutputFormat.NONE:
+                return makePrettyWriter();
```

Comment:
It looks like makePrettyWriter is just a placeholder here because NONE causes nothing to be written. Is that right? If that's the case, is there a more obvious placeholder that could be used (like null)? I think a comment would be helpful, as a minimum.